action.yml

```
@@ -74,17 +105,8 @@ runs:
         COSIGN_PRIVATE_KEY: ${{ inputs.cosign_private_key }}
         GH_TOKEN: ${{ inputs.registry_token }}
         GH_PR_EVENT_NUMBER: ${{ inputs.pr_event_number }}
-        # Uses GitHubs ternary syntax to set cli version, see https://docs.github.com/en/actions/learn-github-actions/expressions#example
-        CLI_VERSION_TAG: ${{ inputs.use_unstable_cli == 'true' && 'main' || 'v0.8.2' }}
+        BB_BUILDKIT_CACHE_GHA: ${{ inputs.use_cache }}
       run: |
-        podman run \
-          -v buildah-vol:/var/run/containerd \
-          -v ${PWD}:/bluebuild \
-          --env-host \
-          --network=host \
-          --privileged \
-          --device /dev/fuse \
-          ghcr.io/blue-build/cli:${CLI_VERSION_TAG}-alpine \
-          build -vv --push ./config/${{ inputs.recipe }} \
-          --registry ${{ inputs.registry }} \
-          --registry-namespace ${{ inputs.registry_namespace }}
+          bluebuild build -vv --push ./config/${{ inputs.recipe }} \
```

```suggestion
          bluebuild build -v --push ./config/${{ inputs.recipe }} \
```

Probably don't want to have trace logging for normal users